Kite SDK/PSPrintSDK/ViewController.m

```diff
@@ -35,6 +38,55 @@ -(void)viewDidAppear:(BOOL)animated{
 - (void)viewDidLoad {
     [super viewDidLoad];
     
+#ifndef DEBUG
+    self.navigationController.navigationBar.hidden = YES;
```

Comment:
You are right. I'll make another app to use for App Store submission to keep this clean(er)